code/tests/test_app.py

```diff
@@ -258,16 +261,32 @@ def setup_method(self):
             ],
         }
 
```

Comment:
Is there a way to extract the env_helper_mock setup to be common for all test cases, and have each test case only override the values if needed? I'm asking because we're needing to write so much env_helper setup for each test case.

Reply:
I've put it into a fixture, what do you think?

Reply:
Amazing, thanks for looking into this!